assets/training/finetune_acft_image/environments/acft_video_mmtracking/context/requirements.txt

```diff
@@ -0,0 +1,17 @@
+azureml-metrics[video]====0.0.31
+azureml-evaluate-mlflow=={{latest-pypi-version}}
+azureml-acft-accelerator=={{latest-pypi-version}}
+azureml-acft-common-components[image]=={{latest-pypi-version}}
+azureml-acft-image-components==0.0.31
+azureml-core==1.52.0
+azure-ai-ml==1.5.0
+requests
+datasets==2.3.2
+optimum==1.6.4
+transformers==4.30.0
```

Comment:
Please check if we can upgrade this and datasets to align with mmdetection environment.

Reply:
transformers upgrade would also not result in vulnerabilities.

Reply:
updated